assets/js/stripe.js

```diff
@@ -531,7 +531,7 @@ jQuery( function( $ ) {
 				return;
 			}
 
-			wc_stripe_form.form.submit();
```

Comment:
Nice cleanup! I can't even find it's deprecated in jQuery's documentation, but I remember reading something about it :)

Reply:
Forgot to mention this was changed in https://github.com/woocommerce/woocommerce-gateway-stripe/issues/1390 but then (seemingly unintentionally) changed back in #1398.